presentation/src/main/java/com/architect/coders/mu8/splash/SplashViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.architect.coders.mu8.splash
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.architect.coders.mu8.BuildConfig
+import com.architect.coders.mu8.utils.Scope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+class SplashViewModel : ViewModel(), Scope by Scope.Impl() {
+
+    companion object {
+        const val TIME_SLEEP: Long = 5000
+    }
+    sealed class UiModel {
```

Comment:
Remember to separate methods/Inner Classes with one line.

Reply:
ok